packages/desktop-client/src/components/budget/rollover/BudgetSummary.js

```diff
@@ -379,10 +379,6 @@ export function BudgetSummary({ month, isGoalTemplatesEnabled }) {
                         name: 'apply-goal-template',
                         text: 'Apply budget template',
                       },
-                      isGoalTemplatesEnabled && {
-                        name: 'overwrite-goal-template',
-                        text: 'Overwrite with budget template',
-                      },
```

Comment:
I felt like this was a useful feature in my testing — why has it been removed?

Reply:
https://github.com/actualbudget/actual/issues/959#issuecomment-1526181423
This is where that started.  It may not be necessary now that certain types of goals get forced to the same priority.  

I would argue that always overwriting is more consistent since all the other options in that menu replace everything always.

Reply:
At the time, i couldn't get it to work properly.  I can try to put it back in and see how it goes.

Reply:
Ok.  I think it's working.

Reply:
The prioritization works by incrementally filling in the budget cells, which is why I needed it to start at 0 each time to get it to work. This was also convenient for checking available balances.  I think if we need to keep the feature of not overwriting the cells, a different approach needs to be implemented so that the budget cells aren't used in the calculations.  From my perspective, it feels like a major overhaul, but maybe you see a shorter path. I'll continue to think about this, but I'm not too hopeful I'll figure it out soon.